pkg/config/config.go

```diff
@@ -395,3 +398,43 @@ func (c *ConfigInfo) DeleteProject(projectName string) error {
 	}
 	return nil
 }
+
+var (
+	// records information on supported parameters
+	supportedParameterDescriptions = map[string]string{
+		"UpdateNotification": "Controls if an update notification is shown or not (true or false)",
```

Comment:
Can this be configurables be string constants?
I see them being referred in more than 1 places which might be prone to bugs
So, this then converts to map[string_alias]string

Reply:
Let me see what I can do.

Reply:
Extracted constants.